### PR TITLE
#64 Update manager's development docs

### PR DESCRIPTION
**Pull request by konekoya**

### CODE REVIEW

File: config/_default/config.toml [hunk 1]
```diff
@@ -34,7 +34,7 @@ removePathAccents = true  # Workaround for https://github.com/gohugoio/hugo/issu
 
 summaryLength = 30  # Listing summary length in words. Also, see `abstract_length` in `params.toml`.
 paginate = 10  # Number of items per page in paginated lists.
-enableEmoji = true
+enableEmoji = false
```
Comment (konekoya) [inline config/_default/config.toml:37]: Disable emoji convertion

File: themes/academic/layouts/shortcodes/alert.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="alert alert-{{ .Get 0 }}">
   <div>
-    {{ .Inner | markdownify | emojify }}
+    {{ .Inner | markdownify }}
```
Comment (konekoya) [inline themes/academic/layouts/shortcodes/alert.html:3]: Should also drop this and so the emoji conversion is completely disabled...

File: content/en/docs/master/manager_dev_guide.md [hunk 3]
```diff
@@ -49,7 +49,6 @@ $ yum install -y xorg-x11-server-Xvfb gtk2-2.24* libXtst* libXScrnSaver* GConf2*
 
 ## Development
 
-
 {{% alert hint %}}
 If this is your first time running this project, you need to complete
 the [Initial machine setup](#initial-machine-setup) section above 👆
```
Comment (konekoya) [inline content/en/docs/master/manager_dev_guide.md:54]: Normal emoji still works

File: content/en/docs/master/manager_dev_guide.md [hunk 4]
```diff
@@ -332,6 +334,30 @@ $ yarn format
...
 
+### Type checking
+
+We're in the progress of moving from plain JavaScript to TypeScript. Although not all source are moved to TS, you can still type checking files:
+
+```shell script
+$ yarn typecheck
+```
+
+### License
```
Comment (konekoya) [inline content/en/docs/master/manager_dev_guide.md:345]: Add missing docs for some npm scripts

File: config/_default/config.toml (hunk 1 above)
Comment (saivirtue) [reply config/_default/config.toml:37]: 👍

**Review by saivirtue — APPROVED** [review]

**Pull request merged**